### DISCUSSION TITLE
Enable support for Azure Functions PowerShell runtime

### DISCUSSION THREAD

**Discussion by pazdedav** [Azure/azure-dev#1422] [Ideas]
Hello,

I asked this question during the January community call.

I see an enormous potential how IT Pros could leverage AZD when designing and implementing 'event-driven process automation' scenarios.

**Comment by craigofnz** [comment]
I read your blog article on this, thanks. I've been using a post provisioning hook for powershell functions but this does not follow the idioms of the azd cli itself.
